doc/reference/networking/lwm2m.rst

```diff
@@ -11,8 +11,281 @@ Overview
 ********
 
 Lightweight Machine to Machine (LwM2M) is an application layer protocol
-based on CoAP/UDP, and is designed to expose various resources for reading,
-writing and executing via an LwM2M server in a very lightweight environment.
+designed with device management, data reporting and device actuation in mind.
+Based on CoAP/UDP, LwM2M is a standard defined by the Open Mobile Alliance.
```

Comment:
Could you add a reference link to the standard here. If it is not public, then wikipedia article might be ok too. Or it might be nice for the user if there would be both the standard link and wikipedia link here. In simplest form it could be just
```
, `LwM2M <https://en.wikipedia.org/wiki/OMA_LWM2M>`_ is a `standard <http://xxxxx>`_ defined .....
```